Update move-resource-group-and-subscription.md (#124641)
Azure/azure-content: articles/azure-resource-manager/management/move-resource-group-and-subscription.md

No, you can't move a resource group to a new subscription. But, you can move all of the resources in the resource group to a resource group in another subscription. Settings such as tags, role assignments, and policies aren't automatically transferred from the original resource group to the destination resource group. You need to reapply these settings to the new resource group. For more information, see [Move resources to new resource group or subscription](./move-support-resources.md).

> ### Unsupported scenarios?
> 
> If resources from Subscription A is migrated to Subscription B and At Same time if resources from Subscription B is migrated to Subscription C then platform blocks this kind of migration. This is by design.

Reviewer: ```suggestion
### Unsupported scenarios

The platform blocks a scenario where resources from Subscription A are migrated to Subscription B when *at the same time* resources from Subscription B are migrated to Subscription C. This is by design.
```
